testing/framework/TestCmd.py

```diff
@@ -1587,7 +1581,8 @@ def run(self, program=None,
         if is_List(stdin):
             stdin = ''.join(stdin)
 
-        if stdin and IS_PY3:#  and sys.version_info[1] < 6:
+        # TODO: was: if stdin and IS_PY3:#  and sys.version_info[1] < 6:
+        if stdin:
```

Comment:
so it was checking for python < 3.6?

Reply:
No, not exactly... someone had coded two places to do that, and then left the "< 6" part commented out. I guess something expected to work didn't work after all... I left the TODO in case we want to swing back and check someday.

Reply:
gotcha